resources/game_config.json

```diff
@@ -69,7 +69,7 @@
 		"max_interaction": 5,
 		"max_interaction_special": 8,
 		"compatibility_effect": 5,
-		"passive_influence_div": 1.3,
+		"passive_influence_div": 1.5,
```

Comment:
Please take a look, that this is not too much of a buff for passive influence

Reply:
Oh, the way it does it, it divides by that number, so a larger passive_influence_div actually results in a smaller amount of passive influence - I was adjusting it down because there will be a lot more passive influence events now. Or have I misunderstood what you mean? 😅😅